src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/cas1/Cas1SpaceBookingController.kt

```diff
@@ -100,7 +100,7 @@ class Cas1SpaceBookingController(
     val user = userService.getUserForRequest()
     val offenderSummaries = offenderService.getPersonSummaryInfoResults(
       crns = searchResults.map { it.crn }.toSet(),
-      limitedAccessStrategy = user.cas1LimitedAccessStrategy(),
+      limitedAccessStrategy = user.casLimitedAccessStrategy(),
```

Comment:
see comments on https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/2401 regarding usage of the strategy

Reply:
change done in [2401](https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/2401)